components/x-follow-button/stories/knobs.js

```diff
@@ -0,0 +1,48 @@
+// To ensure that component stories do not need to depend on Storybook themselves we return a
+// function that may be passed the required dependencies.
+module.exports = (data, { text, boolean, selectV2 }) => {
+	const Groups = {
+		Text: 'Text',
+		Status: 'Status',
+		Flags: 'Flags',
+		ExtraClasses: 'ExtraClasses'
+	};
+
+	const Text = {
+		buttonText () {
+			return text('Default text', data.buttonText, Groups.Text);
+		},
+		alternateText () {
```

Comment:
As in the readme comment - should there also be a `text` knob?

Reply:
Not sure what do you mean here =) (there is a text category of knobs)

Reply:
Sorry I realised what this was later. 

I think it should be called `alternateButtonText` or `altButtontext` to imply it replaces `buttonText`

Reply:
Oh, I see, makes sense